.idea/misc.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="Black">
```

Comment:
Could you please revert these changes? They were probably added automatically by PyCharm.

(Some form of this change should probably be committed separately -- e.g., all `.idea` `xml` files don't have newlines at their EOFs -- but not here.